spark/dl/pom.xml

```diff
@@ -142,6 +142,7 @@
                     <systemProperties>
                         <bigdl.disableCheckSysEnv>true</bigdl.disableCheckSysEnv>
                     </systemProperties>
+                    <argLine>-Xmx6g</argLine>
```

Comment:
@srowen Aha, I know what's happening. It's not caused by the total size of heap. 6GB is enough for me. But my maven is running with JAVA7, so I will get an OOM Error without "-XX:MaxPermSize=1G".
Could you help to add this configuration to your PR?